corehq/apps/hqwebapp/static/hqwebapp/js/rollout_modal.js

```diff
@@ -14,10 +14,10 @@ hqDefine("hqwebapp/js/rollout_modal", [
     googleAnalytics,
     kissmetricsAnalytics
 ) {
+    var secureCookies = initialPageData.get('secure_cookies');
```

Comment:
You can kill this whole file (and `hqwebapp/rollout_modal.html`) if you like, it's not being used. Also fine to leave it alone if you don't want to increase the scope of this PR.

Reply:
I can make another PR after this has been merged to handle that 👍